src/java_binding/src/lib.rs

```diff
@@ -539,13 +540,11 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_rowGetTimestampV
     idx: jint,
 ) -> JObject<'a> {
     execute_and_catch(env, move || {
-        let millis = pointer
-            .as_ref()
-            .datum_at(idx as usize)
-            .unwrap()
-            .into_timestamp()
-            .0
-            .timestamp_millis();
+        let scalar_value = pointer.as_ref().datum_at(idx as usize).unwrap();
+        let millis = match scalar_value {
```

Comment:
Any reason to do this match？

Reply:
previously, we only support sinking risingwave `timestamp` to mysql `timestamp`.
however, now we also need to support sinking risingwave `timestamptz` to mysql `timestamp`, since mysql has no `timestamptz` type.
the `timestamptz` is in the form of `ScalarRefImpl::Int64` in risingwave

Reply:
I have now added the comments..